src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -47,6 +47,12 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
   // have the updates in the correct order.
   const updates = orderBy(release.updates, 'on', 'desc');
 
+  const showAllFilesButton = release.downloadFiles.some(
+    file =>
+      file.type === 'Data' ||
+      (file.type === 'Ancillary' && file.name !== 'All files'),
```

Comment:
I'm just wondering why we do this and if we need to take into account that the 'All files' file type is now 'zip' rather than 'ancillary'. After the recent changes I'm not sure that it's possible to get an 'Ancillary' file with name 'All files'.